test/README.md

```diff
@@ -0,0 +1,52 @@
+# Charts Testing
+
+## Pull Request Testing
+
+### Procedure
+
+Pull requests testing is run via the [Kuberentes Test Infrastructure](https://github.com/kubernetes/test-infra).
+
+The configuration of the Pull Request trigger is [in the config.json](https://github.com/kubernetes/test-infra/blob/827797c54b48295045698465b437f463ca9276c2/jobs/config.json#L10285).
```

Comment:
Should this point to the master version rather than a specific commit for the config? That way it keeps up with the changes and someone can read the latest.

Reply:
It points to a particular block that could move up, down or sideways as they add more code. I pick an explicit commit so that it points to the actual code. I'm not really sure how often this file changes on their end so was trying to be safe.

Reply:
I guess it works for now. Move fast and adapt to change. This works for me.